src/components/RadioButton/index.js

```diff
@@ -0,0 +1,72 @@
+import { Component } from 'pet-dex-utilities';
+import './index.scss';
+
+const events = ['change', 'value:change', 'text:change', 'name:change'];
+
+const html = `
+<label class="radio-container">
+    <input data-select="radio-button" type="radio" name="" value="">
+    <span data-select="radio-button-text" class="radio-dot"></span>
+</label>
+`;
+
+export default function RadioButton({
+  isChecked = false,
+  text = '',
+  value = '',
+  name = '',
+} = {}) {
+  Component.call(this, { html, events });
+
+  this.setChecked(isChecked);
+  this.setText(text);
+  this.setValue(value);
+  this.setName(name);
```

Comment:
Inicializar o estado de desabilitado

```suggestion
  this.setChecked(isChecked);
  this.setText(text);
  this.setValue(value);
  this.setName(name);
  this.setDisabled(isDisabled);
```